src/metabase/sync_database/sync.clj

```diff
@@ -143,6 +143,67 @@
       (catch Throwable t
         (log/error (u/format-color 'red "Unexpected error syncing table") t)))))
 
+(def ^:private ^:const blacklisted-table-names
```

Comment:
Can we call those "technical-table-names" instead of blacklisted? I foresee a world where we might have multiple lists of tables to ignore for different reasons, eg "common join tables", etc.